ledger/sandbox-classic/src/test/suite/scala/platform/sandbox/stores/ledger/TransactionTimeModelComplianceIT.scala

```diff
@@ -62,7 +62,7 @@ class TransactionTimeModelComplianceIT
       case BackendType.InMemory =>
         LedgerResource.inMemory(ledgerId, participantId, timeProvider)
       case BackendType.Postgres =>
-        newLoggingContext { implicit logCtx =>
+        newLoggingContext { implicit loggingContext =>
```

Comment:
See if `LoggingContext.ForTesting` can be used here.

Reply:
Addressed by https://github.com/digital-asset/daml/pull/6924/commits/ef1aa1c2b4892c4482f7c10a35df507750fe5200